doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -1319,6 +1319,13 @@ The issues in this section are related to the :ref:`ug_wifi` protocol.
 
 .. rst-class:: v3-0-0
 
+SHEL-3604: Crash during Wi-Fi SoftAP mode initialization
+  When the Wi-Fi SoftAP sample is built and run, a crash occurs during initialization due to a hostap stack overflow.
+
+  **Workaround:** Increase the hostap stack size to 5800 bytes or cherry-pick changes from PR #2822 in the `sdk-zephyr`_ repository.
+
```

Comment:
Could add the affected platforms.

Reply:
As per offline discussion with @bama-nordic- Affected board is not specified as it is a generic issue and agnostic to the board(s) used for testing.